tests/ut/test_factory.py

```diff
@@ -3,23 +3,38 @@
 import pytest
 
 from aiocache import AIOCACHE_CACHES, Cache, caches
-from aiocache.backends.memcached import MemcachedCache
 from aiocache.backends.memory import SimpleMemoryCache
-from aiocache.backends.redis import RedisCache
 from aiocache.exceptions import InvalidCacheType
 from aiocache.factory import _class_from_string, _create_cache
 from aiocache.plugins import HitMissRatioPlugin, TimingPlugin
 from aiocache.serializers import JsonSerializer, PickleSerializer
 
-assert Cache.REDIS is not None
-assert Cache.MEMCACHED is not None
-CACHE_NAMES = (Cache.MEMORY.NAME, Cache.REDIS.NAME, Cache.MEMCACHED.NAME)
 
+CACHE_NAMES = [Cache.MEMORY.NAME]
 
+try:
+    from aiocache.backends.memcached import MemcachedCache
+except ImportError:
+    MemcachedCache = None
```

Comment:
It would probably be nice to have a test run that does cover this. Can figure it out later though.

Reply:
I'll try to update the test matrix in a minute.